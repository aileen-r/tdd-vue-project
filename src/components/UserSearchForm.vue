<template>
    <div>User Search Form</div>
</template>

<script>
export default {
    name: 'UserSearchForm'
}
</script>

<style>

</style>
