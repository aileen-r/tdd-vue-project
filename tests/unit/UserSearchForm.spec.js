import { shallowMount } from '@vue/test-utils';
import UserSearchForm from '@/components/UserSearchForm';

describe('UserSearchForm', () => {
  const build = () => {
    const wrapper = shallowMount(UserSearchForm);

    return {
      wrapper,
      input: () => wrapper.find('input'),
      button: () => wrapper.find('button')
    };
  };

  it('renders the component', () => {
    const { wrapper } = build();
    expect(wrapper.html()).toMatchSnapshot();
  });
});

// emits "submitted" event on form submission
