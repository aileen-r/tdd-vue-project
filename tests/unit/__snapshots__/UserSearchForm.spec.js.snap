// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserSearchForm renders the component 1`] = `<div>User Search Form</div>`;
